src/main/java/it/gov/pagopa/pu/organization/exception/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package it.gov.pagopa.pu.organization.exception;
+
+import org.springdoc.api.ErrorMessage;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+
+@RestControllerAdvice
+public class ControllerExceptionHandler {
+
+  @ExceptionHandler(ResourceNotFoundException.class)
+  @ResponseStatus(value = HttpStatus.NOT_FOUND)
+  public ErrorMessage resourceNotFoundException(ResourceNotFoundException ex, WebRequest request) {
+    return new ErrorMessage("resource not found: %s".formatted(ex.getMessage()));
```

Comment:
can you log the requested operation as done in
https://github.com/pagopa/p4pa-auth/blob/develop/src/main/java/it/gov/pagopa/payhub/auth/exception/AuthExceptionHandler.java#L92

Reply:
done